ui/raidboss/data/02-arr/dungeon/haukke_manor.js

```diff
@@ -0,0 +1,81 @@
+
+
+[{
+  zoneId: ZoneId.HaukkeManor,
+  triggers: [
+    {
+      // Dark Mist Stun with Suppression for duplicate messages
+      id: 'Haukke Normal Dark Mist Stun',
+      netRegex: NetRegexes.startsUsing({ id: '2C1', source: ['Manor Maidservant', 'Manor Claviger', 'Lady Amandine'] }),
+      condition: function(data, matches) {
+        return data.CanStun();
+      },
+      suppressSeconds: 2,
```

Comment:
Yes, the hallway before the first boss has two Maidservants that like to walk around together, so if one comes you're often getting both. If you don't, the one that is left will often come in with the that's right in front of the first boss door. Im using suppressSeconds to avoid multiple calls and double printing for classes with stuns.